### PR TITLE
new package: youtube-dl-git

### PR DESCRIPTION
**Pull request by a-schaefers**
## Description of package

By the way, I would appreciate help maintaining this one if possible. It updates frequently... Lol
## Checklist

- [x] Latest upstream version.
- [x] I agree to maintain this package (**required**).
- [x] I agree to notify KISS if I can no longer maintain this package (**required**).

### CODE REVIEW

**Comment by dylanaraps** [comment]
Best you just package it as `git` I'd say.

**Comment by a-schaefers** [comment]
done

File: community/youtube-dl/version [hunk 1]
```diff
@@ -1 +1 @@
-2020.01.15 1
+9999 1
```
Comment (dylanaraps) [inline community/youtube-dl/version]: Versions are `git`.

File: community/youtube-dl/sources [hunk 2]
```diff
@@ -1,2 +1,2 @@
-https://github.com/ytdl-org/youtube-dl/archive/2020.01.15.tar.gz
+git+https://github.com/ytdl-org/youtube-dl.git
```
Comment (dylanaraps) [inline community/youtube-dl/sources]: `.git` can be droppd from the end.

**Comment by a-schaefers** [comment]
should be good now

File: community/youtube-dl-git/files/mpv-ytdl-helper.sh [hunk 3]
```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+youtube-dl -o - "$@" - | mpv -
```
Comment (dylanaraps) [inline community/youtube-dl-git/files/mpv-ytdl-helper.sh]: This file can be removed.

Comment (a-schaefers) [reply community/youtube-dl-git/files/mpv-ytdl-helper.sh]: k

**Comment by a-schaefers** [comment]
done

File: community/youtube-dl-git/checksums [hunk 4]
```diff
@@ -0,0 +1,2 @@
+git  git+https://github.com/ytdl-org/youtube-dl.git
+61f9425616d1fe187c8b62e53d6c91a2846e16e21aea7045ddce84b55a8956c2  mpv-ytdl-helper.sh
```
Comment (dylanaraps) [inline community/youtube-dl-git/checksums]: Checksums need updating.

File: community/youtube-dl-git/build [hunk 5]
```diff
@@ -0,0 +1,6 @@
+#!/bin/sh -e
+
+make youtube-dl
+
+install -Dm755 youtube-dl "$1/usr/bin/youtube-dl"
+install -Dm755 mpv-ytdl-helper.sh "$1/usr/bin/mpv-ytdl-helper.sh"
```
Comment (dylanaraps) [inline community/youtube-dl-git/build]: Build needs updating.

**Pull request merged**